2.x: Docs: WLS Integration Code issues

[//]: # "Please remove these comments"
[//]: # "Provide a general summary of the issue in the Title above"

## Environment Details
* Helidon Version:2.5.5
* Helidon SE or Helidon MP: Helidon MP
* JDK version: JDK11
* OS: Linux
* Docker version (if applicable):19.03

----------
We have some document issues with helidon 2.x. Below are the list of issues we found while going through the quick start guide. Please help in correcting the doc.

Issues Identified in Doc:
############################
1. Seeing compilation errors with file and classpath methods in the below script. 
```
    private static Config buildConfig() {
        return Config.builder()
            .sources(
                file("/etc/config/config-file.properties").optional(), 
                classpath("META-INF/microprofile-config.properties")) 
            .build();
    }
```

    Ref page:
    https://helidon.io/docs/v2/#/mp/guides/03_config
    Integration with Kubernetes


3. Custom web context path is not working as per the steps mentioned doc.

    application.yaml content in the resources directory:
    health:
      web-context: "myhealth"  

    Ref page:
    https://helidon.io/docs/v2/#/mp/guides/04_health
    Custom Health Root Path and Port


4. Typo mistakes In Yaml format. 
    Mentioned enables inplace of enabled
  
    Ref page:
    https://helidon.io/docs/v2/#/mp/guides/05_metrics
    Controlling Metrics By Registry Type and Metric Name

5. Annotation issues
    @Path annotation is required along with @Dependent annotation for ExampleResource

    Ref Pag:
    https://helidon.io/docs/v2/#/mp/guides/07_datasource
    Inject a Datasource into Your Application Code

6. Helm repository issues are reported with helm 2.x, and no documentation is available for helm 3.x

    Ref Page:
    https://helidon.io/docs/v2/#/mp/guides/05_metrics
    Prometheus Integration


7. Class loader issues are reported with artifact helidon-integrations-cdi-jta-weld with JTA samples while trying to run the application

    Ref Page:
    https://helidon.io/docs/v2/#/mp/guides/08_jta
    Add the Helidon JTA Integration to Your Application’s Runtime Classpath


    Issue log:
    ----------
    2023.01.05 13:37:47 INFO io.helidon.common.LogConfig Thread[main,5,main]: Logging at initialization configured using classpath: /logging.properties
    Exception in thread "main" java.util.ServiceConfigurationError: Error instantiating :io.helidon.integrations.jta.cdi.NarayanaExtension
          at org.jboss.weld.util.ServiceLoader.createInstance(ServiceLoader.java:315)
    . . . 
    Caused by: java.lang.NoClassDefFoundError: javax/transaction/SystemException
            at java.base/java.lang.Class.getDeclaredConstructors0(Native Method)
    . . . 
    Caused by: java.lang.ClassNotFoundException: javax.transaction.SystemException
            at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
            at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
    . . .

--- a/docs/common/guides/metrics.adoc
+++ b/docs/common/guides/metrics.adoc
@@ -523,8 +523,10 @@ into Kubernetes, then verify that it discovered the Helidon metrics in your appl
 [source,bash]
 .Install Prometheus and wait until the pod is ready:
 ----
-helm install stable/prometheus --name metrics
-export POD_NAME=$(kubectl get pods --namespace default -l "app=prometheus,component=server" -o jsonpath="{.items[0].metadata.name}")
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+helm install metrics prometheus-community/prometheus
+export POD_NAME=$(kubectl get pod -l app.kubernetes.io/name=prometheus -l  app.kubernetes.io/component=server -o jsonpath="{.items[0].metadata.name}")
 kubectl get pod $POD_NAME
 ----
 
@@ -536,12 +538,12 @@ metrics-prometheus-server-5fc5dc86cb-79lk4   2/2     Running   0          46s
 ----
 
 [source,bash]
-.Create a port-forward so you can access the server URL:
+.Create a port-forward, so you can access the server URL:
 ----
 kubectl --namespace default port-forward $POD_NAME 7090:9090
 ----
 
-Now open your browser and navigate to `http://localhost:7090/targets`.  Search for helidon on the page and you will see your
+Now open your browser and navigate to `http://localhost:7090/targets`.  Search for helidon on the page, and you will see your
 Helidon application as one of the Prometheus targets.
 
 ==== Final Cleanup

--- a/docs/mp/guides/07_datasource.adoc
+++ b/docs/mp/guides/07_datasource.adoc
@@ -1,6 +1,6 @@
 ///////////////////////////////////////////////////////////////////////////////
 
-    Copyright (c) 2019, 2021 Oracle and/or its affiliates.
+    Copyright (c) 2019, 2023 Oracle and/or its affiliates.
 
     Licensed under the Apache License, Version 2.0 (the "License");
     you may not use this file except in compliance with the License.
@@ -54,9 +54,7 @@ mvn -U archetype:generate \
     -DarchetypeVersion={helidon-version} \
     -DgroupId=io.helidon.example \
     -DartifactId=helidon-ds \
-    -Dpackage=io.helidon.example.ds \
-    -DrestResourceName=ExampleResource \
-    -DapplicationName=ExampleApplication
+    -Dpackage=io.helidon.example.ds
 ----
 
 Now `cd` into `helidon-ds`.  The rest of this guide will assume all
@@ -72,7 +70,6 @@ Add the following dependency in your `pom.xml`:
 <dependency>
     <groupId>com.h2database</groupId>
     <artifactId>h2</artifactId>
-    <version>1.4.199</version>
     <scope>runtime</scope>
 </dependency>
 ----
@@ -137,11 +134,11 @@ class].
 
 NOTE: For more information about datasources, see https://docs.oracle.com/javase/8/docs/api/javax/sql/DataSource.html
 
-In the `src/main/java/io/helidon/example/ds/ExampleResource.java` file, add the following
+In the `src/main/java/io/helidon/example/ds/GreetResource.java` file, add the following
 imports:
 
 [source,java]
-.`src/main/java/io/helidon/example/ds/ExampleResource.java`
+.`src/main/java/io/helidon/example/ds/GreetResource.java`
 ----
 import javax.enterprise.context.Dependent;
 import javax.inject.Inject;
@@ -152,19 +149,20 @@ import javax.sql.DataSource;
 Annotate the resource class declaration with `@Dependent`:
 
 [source,java]
-.`src/main/java/io/helidon/example/ds/ExampleResource.java`
+.`src/main/java/io/helidon/example/ds/GreetResource.java`
 ----
+@Path("/greet")
 @Dependent <1>
-public class ExampleResource {
+public class GreetResource {
 ----
 
-<1> This ensures that `io.helidon.example.jpa.ExampleResource` is a
+<1> This ensures that `io.helidon.example.jpa.GreetResource` is a
 discoverable CDI bean.
 
 Then add the following annotated field declaration:
 
 [source,java]
-.`src/main/java/io/helidon/example/ds/ExampleResource.java`
+.`src/main/java/io/helidon/example/ds/GreetResource.java`
 ----
 @Inject <1>
 @Named("test") <2>
@@ -185,11 +183,11 @@ Here, the `test` data source is requested.
 
 Now that you have a `DataSource`, you'll use it to connect to the database.
 
-First, ensure the `io.helidon.example.ds.ExampleResource` resource
+First, ensure the `io.helidon.example.ds.GreetResource` resource
 class imports various `java.sql` classes:
 
 [source,java]
-.`src/main/java/io/helidon/example/ds/ExampleResource.java`
+.`src/main/java/io/helidon/example/ds/GreetResource.java`
 ----
 import java.sql.Connection;
 import java.sql.PreparedStatement;
@@ -197,10 +195,10 @@ import java.sql.ResultSet;
 import java.sql.SQLException;
 ----
 
-Add the following resource method to the `ExampleResource` class:
+Add the following resource method to the `GreetResource` class:
 
 [source,java]
-.`src/main/java/io/helidon/example/ds/ExampleResource.java`
+.`src/main/java/io/helidon/example/ds/GreetResource.java`
 ----
 @GET
 @Path("tables")
@@ -252,7 +250,7 @@ Execute the following:
 
 [source,bash]
 ----
-curl http://localhost:8080/example/tables
+curl http://localhost:8080/greet/tables
 ----
 
 Observe that the result will be a list of database table names.

